app/src/org/commcare/android/database/AndroidTableBuilder.java

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ */
+package org.commcare.android.database;
+
+import android.util.Pair;
+
+import org.commcare.android.storage.framework.Table;
+import org.commcare.modern.database.TableBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Some additional table methods we need only on Androidp
+ *
+ * @author ctsims
+ * @author wspride
+ *
+ */
+public class AndroidTableBuilder extends TableBuilder {
+    public AndroidTableBuilder(Class c) {
+        super(c, ((Table) c.getAnnotation(Table.class)).value());
+    }
+
+    public AndroidTableBuilder(String name) {
+        super(name);
+    }
+
+    //TODO: Read this from SQL, not assume from context
+    private static final int MAX_SQL_ARGS = 950;
+
+    public static List<Pair<String, String[]>> sqlList(List<Integer> input) {
```

Comment:
Needs a method level comment since it is not trivial to figure it out by the method name or glancing at the code